inst/report/comparison_report.Rmd

```diff
@@ -183,23 +183,37 @@ if (print_pt) {
 
 :::
 
+::: {#translate lang="fr"}
+
+### Méthodes
+[Naomi](https://github.com/mrc-ide/naomi) est un modèle d'estimation à petite échelle permettant d'estimer la prévalence du VIH et des PVVIH, la couverture TARV et les nouvelles infections par le VIH au niveau du district par sexe et par groupe d'âge de cinq ans. Le modèle combine des données au niveau du district sur des résultats multiples provenant de plusieurs sources dans un modèle statistique bayésien pour produire des indicateurs robustes de la charge du VIH au niveau sous-national.
+
+:::
+
+::: {#translate lang="pt"}
+
+### Métodos
+[Naomi](https://github.com/mrc-ide/naomi){target="_blank"} é um modelo de estimativa de pequena área para estimar a prevalência de VIH e o número de PVVIH, cobertura de TARV e novas infeções por VIH a nível distrital por género e grupo etário de cinco anos. O modelo combina dados de nível distrital sobre resultados múltiplos de várias fontes num modelo estatístico Bayesiano para produzir indicadores robustos da carga subnacional do VIH.
+
+:::
+
 
 
 ```{r, echo = FALSE, results = 'asis'}
 
 if (print_en) {
-  cat("### Comparison of Naomi model estimates to data inputs  \n")
+  cat("### Comparison of Naomi model estimates to data inputs \n")
   cat("\n### Household Survey \n")
 }
 
 if (print_fr) {
-  cat("### Comparison of Naomi model estimates to data inputs  \n")
-  cat("\n### Household Survey \n")
+  cat("### Comparaison des estimations du modèle Naomi avec les entrées de données\n")
+  cat("\n### Données d’enquête auprès des ménages \n")
 }
 
 if (print_pt) {
-  cat("### Comparison of Noami model estimates to data inputs  \n")
-  cat("\n### Household Survey \n")
```

Comment:
Same as above, I don't think we asked UNAIDS about this?

Reply:
Yeah - sorry these got left off I'll follow up.